lib/server/swarm.js

```diff
@@ -29,15 +29,7 @@ function Swarm (infoHash, server) {
       peer_id: peer.peerId
     }
     self._onAnnounceStopped(params, peer, peer.peerId)
-
-    // When a websocket peer is evicted, and it's not in any other swarms, close
-    // the websocket to conserve server resources.
-    if (peer.socket && peer.socket.infoHashes.length === 0) {
-      try {
-        peer.socket.close()
-        peer.socket = null
-      } catch (err) {}
-    }
+    peer.socket = null
```

Comment:
This entire section isn't actually necessary, I realized. It is okay for a client to have a websocket open to the server but to not be in any swarms. Because they might join another swarm in the future.

When the client first connects, they are not in any swarms and we are okay with that. So, there's no reason to close it preemptively here. We always do cleanup once the client closes the socket.